stable/jenkins/templates/deprecation.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.Master }}
+   {{ fail "Master.* values have been renamed, please check the documentation" }}
+{{- end }}
+
+{{- if .Values.Agent }}
+   {{ fail "Agent.* values have been renamed, please check the documentation" }}
+{{- end }}
+
+{{- if .Values.Persistence }}
+   {{ fail "Persistence.* values have been renamed, please check the documentation" }}
+{{- end }}
+
+{{- if .Values.NetworkPolicy }}
+   {{ fail "NetworkPolicy.* values have been renamed, please check the documentation" }}
+{{- end }}
```

Comment:
Missing new line?

Reply:
And maybe we should add something similar for the rbac changes?

Reply:
added checks for all values described in `README.md` which have been renamed. That should avoid a lot of unnecessary questions / issues.